tests/contract-playground/src/MisusedBoolean.sol

```diff
@@ -0,0 +1,37 @@
+pragma solidity 0.4.22;
+
+contract MisusedBoolean {
+
+    function isEven(uint256 num) internal returns(bool) {
+        return num % 2 == 0;
+    }
+
+    function misuse(uint256 num) external returns(uint256) {
+        if (isEven(num) || true) {
+            return num * num;
+        }
+        return 0;
+    }
+
+    function misuse2(uint256 num) external returns(uint256) {
+        if (isEven(num) && false) {
+            return num * num;
+        }
+        return 0;
+    }
+
+    function misuse3(uint256 num) external returns(uint256) {
+        if (false && isEven(num)) {
+            return num * num;
+        }
+        return 0;
+    }
+
+      function misuse4(uint256 num) external returns(uint256) {
+        if (true || isEven(num)) {
+            return num * num;
+        }
+        return 0;
+    }
```

Comment:
What about this case, [from here](https://github.com/crytic/slither/wiki/Detector-Documentation#misuse-of-a-boolean-constant)?

```sol

if (false) { // bad

```

(and equivalent `true` case)